src/Maestro/Maestro.MergePolicies/AllChecksSuccessfulMergePolicy.cs

```diff
@@ -33,7 +33,7 @@ public override async Task<MergePolicyEvaluationResult> EvaluateAsync(IPullReque
 
             if (!notIgnoredChecks.Any())
             {
-                return Fail("Waiting for checks.");
+                return Pending("Waiting for checks.");
```

Comment:
Pending will still block an auto-merge, ya?

Reply:
We would be in this situation so I think yes, it wont auto-merge
![image](https://user-images.githubusercontent.com/43049559/91912892-8c47b700-ecb4-11ea-973a-e8e2b87d2667.png)